src/composer.json

```diff
@@ -26,6 +26,7 @@
         "symfony/runtime": "6.1.*",
         "symfony/security-bundle": "6.1.*",
         "symfony/twig-bundle": "6.1.*",
+        "symfony/validator": "6.1.*",
```

Comment:
Where validator is need for this ticket ? @MisterAzix

Reply:
It is needed when u use `make:crud` command, then it used in the form builder to validate images